ravenpy/utilities/geo.py

```diff
@@ -0,0 +1,238 @@
+import json
+import logging
+from pathlib import Path
+from typing import Union, List
+
+try:
+    import fiona
+    import rasterio
+    import rasterio.mask
+    import rasterio.vrt
+    import rasterio.warp
+    from pyproj import CRS
+    from shapely.geometry import GeometryCollection, shape, Polygon, MultiPolygon, mapping
+    from shapely.ops import transform
+except (ImportError, ModuleNotFoundError) as e:
+    msg = (
+        f"`{Path(__file__).stem}` requires installation of the RavenPy GIS libraries. These can be installed using the"
+        " `pip install ravenpy[gis]` recipe or via Anaconda (`conda env -n ravenpy-env -f environment.yml`)"
+        " from the RavenPy repository source files."
```

Comment:
Maybe it would be more practical to put this particular and very specialized error message somewhere unique, maybe in `utilities.__init__.py`? I mean just the error message itself, to be clear.